test/src/helper/spawn.ts

```diff
@@ -96,11 +108,15 @@ export default class CodeChain {
         });
 
       const readline = createReadline({ input: this.process!.stderr });
+      let flag = false;
       readline.on("line", (line: string) => {
         if (line.includes("Initialization complete")) {
-          readline.close();
+          flag = true;
           resolve();
         }
+        if (this.logFlag && flag) {
+          appendFileSync(this.logPath, line + "\n");
```

Comment:
Why don't you use just `appendFile` here?

Reply:
I thought synchronous appending would be safe in there at that time. I'll fix it.